src/components/SkillListTest.js

```diff
@@ -0,0 +1,176 @@
+import React from "react";
```

Comment:
This doesn't look like you're testing the skillList.js page. Do we need this file?